dataflow/run_template/main_test.py

```diff
@@ -34,6 +34,8 @@
 import main
 
 
+RETRY_MAX_TIME = 5 * 60 # 5 minutes in seconds
```

Comment:
(to appease the linter)
```suggestion
RETRY_MAX_TIME = 5 * 60  # 5 minutes in seconds
```

Reply:
Thank you, I used the online editor so I didn't run the linter locally. I didn't think I would get any issues for such a small change :)